public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, InfoBox, Portal, stylesFactory, useTheme } from '@grafana/ui';
+import { getModalStyles } from '@grafana/ui/src/components/Modal/getModalStyles';
+import { css, cx } from 'emotion';
+
+const getStyles = stylesFactory(() => {
+  return {
+    infobox: css`
+      margin-bottom: 0;
+    `,
+    text: css`
+      margin: 8px 0 16px;
+    `,
+    backdrop: css`
+      background-color: #0b0d10;
+      opacity: 0.8;
+    `,
+  };
+});
+
+interface Props {
+  maxConcurrentSessions?: number;
+}
+
+export const TokenRevokedModal = (props: Props) => {
+  const theme = useTheme();
+
+  const styles = getStyles();
+  const modalStyles = getModalStyles(theme);
+
+  const showMaxConcurrentSessions = props.maxConcurrentSessions && props.maxConcurrentSessions > 0;
```

Comment:
Nit: This can be simplified.
```suggestion
  const showMaxConcurrentSessions = Boolean(props.maxConcurrentSessions);
```

Reply:
Wow, nice trick! 🔝  Thanks!